guides/security/acceptable_use_policy.md

```diff
@@ -0,0 +1,76 @@
+
+## I will ensure my laptop isn’t left unattended or unlocked
+Lock your screen or put your computer to sleep when you’re not around. Colleagues might not have the same permissions as you and visitors might be wandering around where you’re working. When not in use ‘at work’, you can store in the laptop lockers provided or ‘at home’, in a safe place where it can’t be seen. If you are using your device in a public place, don’t leave it unattended and be wary of those who may be watching over your shoulder. 
+
+## I will ensure I report the arrival of guests and visitors promptly
+We keep a record of all visitors to the Made Tech office. All visitors should be logged using the appropriate system and you should alert the member of the team they are here to visit. If someone tries to enter the office and you feel uncomfortable, you are within your rights to challenge their reasons for entry (gently).
+
+## If my device is stolen or missing, I immediately report it
+Lots of information can be extracted from stolen or misplaced laptops, even in some cases when the disk has been encrypted. Report a missing machine quickly, better safe than sorry. We use remote device management to remotely wipe data from your device in the event that it is stolen. You should report all stolen items to the IMS Manager or CSO, who will record the incident. 
+
+## I will backup my work in the right place
+Information saved on a device is at risk of loss or theft and will not be backed up. To protect against theft, laptops are encrypted. Made Tech provides back up facilities within Google (Drive). Each team at Made Tech has their own Team Drive and it is the responsibility of those teams to maintain what is kept in those drives. All data must be copied to these locations to ensure there is no data loss in the event of your laptop becoming unusable.
+
+## I follow the guidance around passwords
+Keep passwords secure and do not share accounts with anyone else. You are responsible for the security of your passwords and account.
+
+## I will not share my laptop with anyone else, including under a separate user account
+The laptop you use for working should not be used by anyone else; for example a partner having a separate user account or sharing a generic login.
+
+## I will not open unfamiliar email attachments, and I will always check the actual address for links in emails, even from trusted colleagues
+Phishing and in particular spear phishing (which targets specific individuals within a specific organisation) is a genuine threat with recorded attacks everyday.
+
+## I report anything that could be a security incident
+ Better safe than sorry. If you experience or see anything suspicious,or anything that you know breaks one of our security policies, report it to either the IMS manager or CSO. As a general rule, if you are not sure whether to report something, report it anyway. We'd always rather know. 
```

Comment:
The second `better safe than sorry` in this doc. A needed Colinism?

Reply:
Also missing a space after a comma

Reply:
Actually it's a me-ism. Makes me sound like my mother. Removed.